### ISSUE TITLE
The MediaPlayer controller widgets position incorrect after commit 4011

### ISSUE THREAD

**Issue by halx99** [axmolengine/axmol#2114]
- axmol version: https://github.com/axmolengine/axmol/commit/e6174ff62f3db7d0df3a9d37044ebdd22e223463
- devices test on:
- developing environments
   - NDK version: r23c
   - Xcode version: 14.2+
   - Visual Studio: 
      - VS version: 2022 (17.9+)
      - MSVC version: 19.39+
      - Windows SDK version: 10.0.22621.0+
   - cmake version: 
Steps to Reproduce:

1. Download: https://github.com/axmolengine/axmol/actions/runs/10578037315/artifacts/1859505076
2. Goto test case:
![image](https://github.com/user-attachments/assets/eb7bc009-828f-4bfd-9505-1382bf2b09d2)

prev version commit 4010 works well: 
https://github.com/axmolengine/axmol/actions/runs/10577457012/artifacts/1859340560

Relative PR: #2102

**Comment by halx99** [comment]
@smilediver could you take a look?

**Comment by smilediver** [comment]
Ok, I'll have a look.

**Comment by smilediver** [comment]
I've tested the artifact, as well as b436da431f06747b7ffba1f07236914a38e7321b, e6174ff62f3db7d0df3a9d37044ebdd22e223463 and latest 7025b1e6fc63951943eb4e4ce8598f1dc59b9076 builds and everything seems to work correctly. This is what I see:

![image](https://github.com/user-attachments/assets/c37868eb-e5ed-4fb4-b7cb-8d930a58502f)

**Comment by halx99** [comment]
you shoud click the video to ensure media controller display

**Comment by rh101** [comment]
Just FYI, the media player controller is positioned using normalized positioning, so it would have been affected by PR #2102, but this was subsequently fixed in PR #2110.

Testing this on Windows resulted in this correct output:
![image](https://github.com/user-attachments/assets/ea15c1c6-3d79-4880-8c6e-3110403d7f56)

@halx99 the screenshot in your initial post is exactly the problem I encountered with `ui::Widget` nodes when using normalized positioning, but as far as I can tell, it has been resolved by PR #2110.  Is it still not working for you?
